plugins/woocommerce/e2e/tests/merchant/create-simple-product.spec.js

```diff
@@ -65,9 +65,21 @@ test.describe( 'Add New Simple Product Page', () => {
 	test( 'can create simple virtual product', async ( { page } ) => {
 		await page.goto( 'wp-admin/post-new.php?post_type=product' );
 		await page.fill( '#title', virtualProductName );
-		await page.click( '#_virtual' );
 		await page.fill( '#_regular_price', productPrice );
+		await page.click( '#_virtual' );
 		await page.click( '#publish' );
+		await page.waitForLoadState( 'networkidle' );
+
+		// When running in parallel, clicking the publish button sometimes saves products as a draft
+		if (
```

Comment:
As the comment states. This check is to handle the occasion when the publish button is pressed, but the product ends up being saved as a draft. I cannot reproduce this manually so it seems to be a PW specific issue. It is worth highlight though